jax/_src/nn/functions.py

```diff
@@ -844,24 +871,29 @@ def dot_product_attention(
   :code:`probs` as the output of :math:`softmax`.
 
```

Comment:
Would it make sense to give the arxiv references to MHA, GQA and MQA? We use these terms, but its useful for users to know what they mean precisely.

Reply:
Sure. Added the paper urls.